### PR TITLE
Add support for Sidekiq 5

### PR DESCRIPTION
**Pull request by maltoe**
Hey guys,

this was easier than thought.

Cheers!

---

I'm trying to convince my coworkers to use this little precious for our recurrent tasks again. For that to work, I'm planning to invest some time here and there: 

........... [snip] Had some text here, but I'm going to put it in issues instead

### CODE REVIEW

File: sidekiq-repeat.gemspec [hunk 1]
```diff
@@ -14,11 +14,12 @@ Gem::Specification.new do |spec|
   spec.files         = Dir['lib/**/*rb']
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sidekiq', '>= 4', '< 5.0'
+  spec.add_dependency 'sidekiq', '>= 4'
```
Comment (dsander) [inline sidekiq-repeat.gemspec]: Should we limit that to `< 6`, at least until #6 is fixed?

File: test/test_helper.rb [hunk 2]
```diff
@@ -124,7 +124,7 @@ def setup
       Sidekiq::Repeat.configure { |config| configure(config) }
 
       @boss = MiniTest::Mock.new
-      2.times { @boss.expect(:options, {:queues => ['default'] }, []) }
+      3.times { @boss.expect(:options, {:queues => ['default'] }, []) }
```
Comment (dsander) [inline test/test_helper.rb]: It surprises me that this works for both versions, doesn't minitest verify the amount of times the method is actually called?

Comment (maltoe) [reply test/test_helper.rb]: No, the minitest API is a bit surprising in that sense. `expect` simply sets up call handling and return values, if you actually want to verify anything, you need to use `verify`.

[Minitest::Mock doc](http://ruby-doc.org/stdlib-2.0.0/libdoc/minitest/rdoc/MiniTest/Mock.html#method-i-verify)

File: sidekiq-repeat.gemspec (hunk 1 above)
Comment (maltoe) [reply sidekiq-repeat.gemspec]: done, didn't know that was a thing yet

Comment (dsander) [reply sidekiq-repeat.gemspec]: I am sure it will be in the future 😄

**Pull request merged**